content/src/Blip.Api.Template/MySettings.cs

```diff
@@ -12,6 +14,30 @@ public class MySettings
         /// </summary>
         [JsonProperty("BlipBotSettings")]
         public BlipBotSettings BlipBotSettings { get; set; }
+
+        [JsonProperty("ContentProvider")]
+        public ContentProviderSettings ContentProvider { get; set; }
+
+        [JsonProperty("SmallTalksConfig")]
+        public Dictionary<string, List<string>> SmallTalksConfig { get; set; }
```

Comment:
This should be removed alonside any mention of the SmallTalks API, and moved to an independent package

As much as smalltalks api is open source (https://github.com/iperoyg/smalltalks), its hosting isnt open and therefore it should not be on this template by default